### ISSUE TITLE
Methodology precision

### ISSUE THREAD

**Issue by FrancoisPgm** [brainhack-school2020/stephaniealley_bhs2020_project#5]
Great idea, I would love to see how much the choice of the confounds will alter the results. However I am not sure I get how you will proceed, will you train one model on data with one kind of preprocessing and then check how it performs on data with an other kind of preprocessing, or will you train two models, one on each kind of preprocessing ? 
I guess you can do both, but I suppose it would entail different interpretations.

**Comment by stephaniealley** [comment]
Right, that is a great point. My initial idea was to train two models, one on each kind of preprocessing. The former approach might be an interesting way to assess the robustness of the model, though. I will take this into consideration.
